apps/aeutil/lib/serialization.ex

```diff
@@ -212,8 +212,8 @@ defmodule Aeutil.Serialization do
   @doc """
   Initializing function to the recursive functionality of deserializing a strucure
   """
-  @spec deserialize_value(any()) :: any()
-  def deserialize_value(value), do: deserialize_value(value, "")
+  @spec deserialize_value(any(), atom()) :: any()
```

Comment:
Fix the specifications of the deserialize_value() and move each specification to it's own function.